src/frontend/src/lib/components/dapps/DappTags.svelte

```diff
@@ -11,8 +12,8 @@
 	class="flex list-none flex-wrap gap-2"
 >
 	{#each tags as tag}
-		<li class="rounded border border-light-grey bg-dust/30 px-2 py-0.5 text-xs font-semibold">
+		<Badge>
```

Comment:
This should have been another PR. Anyway, see other comment, it's not a `li` anymore.